CHANGES.md

```diff
@@ -39,6 +39,7 @@
 * New AnnotateImage & AnnotateImageWithContext PTransform's for element-wise & batch image annotation using Google Cloud Vision API. (Python) ([BEAM-9247](https://issues.apache.org/jira/browse/BEAM-9247))
 * Added a PTransform for inspection and deidentification of text using Google Cloud DLP. (Python) ([BEAM-9258](https://issues.apache.org/jira/browse/BEAM-9258))
 * Added ability to write to BigQuery via Avro file loads (Python)([BEAM-8841](https://issues.apache.org/jira/browse/BEAM-8841))
+* Python SDK will now use Python 3 type annotations as pipeline type hints. ([#10717](https://github.com/apache/beam/pull/10717))
```

Comment:
Please add a link to some docs, and make sure they have a good pointer for how to disable this if it breaks users (that may have bad typing hints and need a quick fix).

Reply:
I'm working on a blog post. We could wait on that (no PR yet), or settle on this PR's description.

Reply:
I've update the description.